stl.go

```diff
@@ -880,3 +951,89 @@ func encodeTextSTL(i string) (o []byte) {
 	}
 	return
 }
+
+func parseJustificationAttribute(i byte) Justification {
+	switch i {
+	case 0x00:
+		return JustificationUnchanged
+	case 0x01:
+		return JustificationLeft
+	case 0x02:
+		return JustificationCentered
+	case 0x03:
+		return JustificationRight
+	default:
+		return JustificationUnchanged
+	}
+}
+
+func isTeletextControlCode(i byte) (b bool) {
+	return i <= 0x1f
+}
+
+func parseOpenSubtitleRow(i *Item, d decoder, fs func() styler, row []byte) error {
+	// Loop through columns
+	var l = Line{}
+	var li = LineItem{InlineStyle: &StyleAttributes{}}
+	var s styler
+	for _, v := range row {
+		// Create specific styler
+		if fs != nil {
+			s = fs()
+		}
+
+		if isTeletextControlCode(v) {
+			return fmt.Errorf("teletext control code in open text")
```

Comment:
I'd rather you use `errors.New("teletext control code in open text")` as you mentioned in your previous comment

Reply:
Done.